test/fixtures/lightscript/if-expression/oneline-then-block/options.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this test should be changed to:

```js
x = 
  if false: 1
  elif true:
    2
```
and should still pass. 

An additional test should be added with the name `over-indented-elif` (or similar) that fails with this error message.